setup.cfg

```diff
@@ -23,15 +23,16 @@ install_requires =
     indexed_gzip >=0.8.8
     nibabel >=2.2.1
     nilearn !=0.5.0, !=0.5.1
-    nipype >=1.2.0
```

Comment:
Are any updates to setup.cfg strictly necessary for this PR? I assume they aren't.

Reply:
reverted them back